test/unit/jwt-token-manager.test.js

```diff
@@ -33,7 +34,7 @@ describe('JWT Token Manager', () => {
 
       const decodeSpy = jest
         .spyOn(jwt, 'decode')
-        .mockImplementation(token => ({ iat: 0, exp: 100 }));
+        .mockImplementation(token => ({ iat: 10, exp: 100 }));
```

Comment:
Can't use `iat = 0` because that triggers the new check in the `jwt-token-manager`.